src/machines/reviewTask/reviewTaskMachine.ts

```diff
@@ -168,6 +170,14 @@ export const transitionHandler = (state) => {
             })
             .finally(() => resetIsLoading());
     }
+
+    sendReviewNotifications(
```

Comment:
suggestion (non-blocking): if you do this in the backend it's more secure and you would still have access to the needed information. This would also reduce the amount of responsibility of this component.